components/p3a/constellation/rs/cxx/Makefile

```diff
@@ -0,0 +1,13 @@
+all: example
```

Comment:
This Makefile does not belong in brave-core

Reply:
i remembered removing this... must have come back up after a rebase or something